web-app/js/campaigns/contest.js

```diff
@@ -69,7 +69,7 @@ $(function () {
     $(".call-to-action").on("click", "a.btn.ADDING_APPLICATIONS", contestApplicationHelper.bindActionClickAddDistrictProposal);
 
     $(".actions.call-to-action-mobile.go-to-action button").on("click", function(e){
-        $(".actions.clearfix a")[0].click()
+        $(".actions.clearfix a")[1].click()
```

Comment:
Pufff. Too risky.  What happen if in the futture other place has an "actions" section wiht an "a"?
Change it for: 
$("#aside-ppal > .call-to-action:first-child > .actions > a.btn")